include/tvm/relay/attrs/nn.h

```diff
@@ -421,9 +421,9 @@ struct Conv3DTransposeAttrs : public tvm::AttrsNode<Conv3DTransposeAttrs> {
             "dimensions respectively. Convolution is applied on the 'D', 'H' and"
             "'W' dimensions.");
     TVM_ATTR_FIELD(kernel_layout)
-        .set_default("OIDHW")
+        .set_default("IODHW")
```

Comment:
Is this the only cases to use `IOxxx` instead of `OIxxx` for conv? like conv2d/conv2d_trans/conv3d?

Reply:
This is the only cases to use `IOXXX` for `conv2d_trans`, `conv3d_trans`.
For `conv2d` and `conv3d`, the corresponding default layout is `OIXXX`.

Reply:
> This is the only cases to use `IOXXX` for `conv2d_trans`, `conv3d_trans`. For `conv2d` and `conv3d`, the corresponding default layout is `OIXXX`.

Ok, thanks!

Reply:
Yes, conv2d_trans's default layout is `IOXX`